server/catgenome/src/main/java/com/epam/catgenome/manager/externaldb/HttpDataManager.java

```diff
@@ -83,7 +85,7 @@ public String fetchData(String locationStub, ParameterNameValue[] params)
             throws ExternalDbUnavailableException {
 
         final String location = getLocationStub(locationStub, params);
-
+        log.info(location);
```

Comment:
Seems to be a debug logging, maybe we don't need it in this PR?

Reply:
Fixed, removed it